File: hard/password/password.py
def validate(password):

	caracters = ('!', '@', '#', '$', '%', '&', '*')

	cont_number = 0
	cont_caracters = 0

	for i in password:
		if i in caracters:
			cont_caracters += 1
		elif i.isdecimal():
			cont_number += 1


	if cont_number >= 2 and cont_caracters >= 2 and len(password) >= 7:
		print("Strong")
	else:
		print("Weak")



password = input()

validate(password)

File: hard/password/password.php
<!DOCTYPE html>
<html>
<head>
	<title>Password</title>
</head>
<body>
	<form  method="post">
		<label for="password">Password : </label>
		<input type="text"  name="password">

		<input type="submit" value="Action" >
	</form>
	<br>
	<?php

		function valid_password($password){

			if ($password == NULL ){
				return("");
			}

			$nums = 0;
			$s_characters = 0;
			$special_characters = Array('!', '@', '#', '$', '%', '&', '*');
			if (strlen($password) < 7 ){
				echo "Weak";
				return null;
			}

			for ($i=0; $i < strlen($password); $i++) { 
				if(is_numeric($password[$i])){
					$nums += 1;
				}
				else if(array_search($password[$i],$special_characters) != false){
					$s_characters += 1;

				}
				if($nums == 2 && $s_characters == 2){
					break;
				}

			}
			if($nums == 2 && $s_characters == 2){
				echo "Strong";
			}else{
				echo "Weak";
			}
			
		}

	    $password = $_POST["password"];

	    valid_password($password);




   
	?>

</body>
</html>
		
